Sources/HeroDebugView.swift

```diff
@@ -23,7 +23,7 @@
 import UIKit
 
 #if os(iOS)
-protocol HeroDebugViewDelegate {
+protocol HeroDebugViewDelegate: class {
```

Comment:
@lkzhaoSC Because of this (https://github.com/realm/SwiftLint/blob/master/Source/SwiftLintFramework/Rules/ClassDelegateProtocolRule.swift#L20).